### DISCUSSION TITLE
Day 11 - here we go!

### DISCUSSION THREAD

**Discussion by CarolineMorton** [CarolineMorton/AdventOfCode_WIR#14] [Ideas]
🎄🎄 Day 11 🎄🎄

How are you all getting on?

**Comment by AR2202** [comment]
Part 1 was fine, but spent way too much time trying to solve part 2!

**Comment by Sowmiya07** [comment]
[Here](https://github.com/Sowmiya07/AdventOfCode_WIR/blob/main/day_11/src/main.rs) is my Day 11 Part 1 and Part 2. Part 2 solution takes 20 seconds for 25 blinks and it runs for a long time for 75. I am trying to optimize.

**Reply by AR2202** [reply]
Well, Mine didn't finish at all, my attempts were either too slow or ran out of memory. Well done for solving it.

**Reply by Sowmiya07** [reply]
Yea, I have also struggled with the same issue. After lot of time, I was able to finally run 75 blinks in 247ms

**Reply by AR2202** [reply]
that's impressive.

**Comment by Sowmiya07** [comment]
Sorry I had to post here as there is no discussion created for Day 13. [Here](https://github.com/Sowmiya07/AdventOfCode_WIR/blob/main/day_13/src/main.rs) is my Day 13 Part 1 solution. Working on Part 2.

**Reply by AR2202** [reply]
Ah nice!
Tried that one in Rust as well, but I have a bug, it works on the example but not the actual input.
[Day13](https://github.com/AR2202/AdventOfCode_WIR/blob/main/day_13/src/main.rs)
UPDATE: I fixed it. It was a problem with numerical precision.
